communication/src/logging.rs

```diff
@@ -1,7 +1,7 @@
 //! Configuration and events for communication logging.
 
 /// Configuration information about a communication thread.
-#[derive(Abomonation, Debug, PartialEq, Eq, Hash, Clone, Copy)]
+#[derive(Debug, PartialEq, Eq, Hash, Clone, Copy)]
```

Comment:
Shouldn't we derive `Serialize`/`Deserialize` here?

Reply:
Good point. I didn't check that any more than `cargo test` passed, so it's very possible much has been missed. It looks like just a scan for "removed `Abomination`, but doesn't have serde" would be valuable. I'll do that!